assemblers/org.osbuild.tar

```diff
@@ -46,8 +46,27 @@ def main(tree, output_dir, options):
             return 1
         extra_args.append(f"--{compression}")
 
-    subprocess.run(["tar", *extra_args, "-cf", f"{output_dir}/{filename}", "-C", tree, "."],
-                   stdout=subprocess.DEVNULL, check=True)
+    tar_env = {}
+    # Speed up xz by allowing it to use all CPU cores for compression.
+    if compression == "xz":
+        tar_env['XZ_DEFAULTS'] = "--threads 0"
```

Comment:
My bad: it's actually `XZ_OPT` (why is only one of them in plural?). It should not really matter which one we're using, because we're running this in a container with full control anway.

Reply:
How about dropping the conditional and always setting the options? Do we win something by using the conditional? I mean, the variables are properly namespaced, and less conditionals always makes easier debugging. A static environment is easier to argue with, than one specialized for a use-case.

Reply:
@dvdhrm I'm happy to drop the conditional and just apply it all the time. It won't have any affect on other compression mechanisms that aren't looking for the environment variable.